client/search/search-input.tsx

```diff
@@ -63,7 +44,7 @@ export const SearchInput = React.forwardRef<SearchInputHandle, SearchInputProps>
           inputRef.current?.focus()
           inputRef.current?.select()
           return true
-        } else if (event.code === ESCAPE && inputFocusedRef.current) {
+        } else if (event.code === ESCAPE) {
```

Comment:
Doesn't this make all escape presses get eaten here, regardless of where their focus is on the page? I'm not sure that's really a great thing to be doing